include/nbl/ui/KeyCodes.h

```diff
@@ -1,26 +1,133 @@
 
 namespace nbl::ui
 {
-	enum E_KEYCODE
-	{
-		// TODO
-	};
-	enum E_KEYMODIFIER
+	enum E_KEY_CODE : uint32_t
```

Comment:
wouldn't it be better to record the key as `char utf8[4];` ?

like where EKC_0 is literally the `0` character?

then only leave a special enum for special keys that dont produce characters (or produce them in a special way)


OR


at the very least ensure that `EKC_0='0'`, etc. / provide a E_KEY_CODE to char function

Reply:
Probably will go with the second option, cause the symbols for `EKC_num` and `EKC_NUMPAD_num` should be the same

Reply:
Btw, where do i put the convertion function?

Reply:
any header in the `ui` namespace with win32 at the end of the name

Reply:
> 
> 
> > provide a E_KEY_CODE to char function
> 
> Probably will go with this option, cause the symbols for `EKC_num` and `EKC_NUMPAD_num` should be the same

btw if you look at the hex codes for letter keys and simple numbers you'll see their values match with the ASCII values for a `char` representing the same value.